cmd/tacod/main.go

```diff
@@ -26,7 +28,12 @@ func main() {
 }
 
 func createServer(rt *taco.Runtime) *restapi.Server {
-	server := restapi.NewServer(handlers.BuildAPI(rt))
+	api := handlers.BuildAPI(rt)
+	server := restapi.NewServer(api)
+	handler := alice.New(
```

Comment:
alice just allows us to handle middleware to swagger connection better? (i.e. avoid the go-swagger config file hookup) - am i understanding that correctly? sorry if mentioned in a ticket i missed.

Reply:
Yeah, alice is a better middleware library (also used by the kvstore app).  We're setting up our middleware outside the config_taco.rb so that we can continue to blow away the whole generated directory when we want to regenerate (see https://github.com/go-openapi/kvstore/issues/3)